shared/feature/feeding/src/debug/java/com/epmedu/animeal/feeding/data/repository/FeedingPointRepositoryMock.kt

```diff
@@ -0,0 +1,81 @@
+package com.epmedu.animeal.feeding.data.repository
+
+import com.epmedu.animeal.common.domain.wrapper.ActionResult
+import com.epmedu.animeal.feeding.domain.model.Feeder
+import com.epmedu.animeal.feeding.domain.model.FeedingPoint
+import com.epmedu.animeal.feeding.domain.model.enum.AnimalState
+import com.epmedu.animeal.feeding.domain.repository.FavouriteRepository
+import com.epmedu.animeal.feeding.domain.repository.FeedingPointRepository
+import com.epmedu.animeal.feeding.presentation.model.MapLocation
+import com.epmedu.animeal.foundation.switch.AnimalType
+import kotlin.random.Random
+import kotlin.random.nextInt
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+internal class FeedingPointRepositoryMock(
+    dispatchers: Dispatchers,
+    private val favouriteRepository: FavouriteRepository
+) : FeedingPointRepository {
+
+    private val stubData = List(25) { index ->
+        FeedingPoint(
+            id = index.toString(),
+            title = "$index - Near to Bukia Garden M.S Technical University",
+            description = "$index - This area covers about 100 sq.m. -S, it starts with Bukia Garden " +
```

Comment:
As a suggestion: You can use a description which includes tags as well, taken from FeedingPointSheetPreview:
                description = "Ordered list : <ol> <li>first item</li> <li>second item</li> " +
                    "<li>third item</li> </ol><h1> Header1 </h1> <h2> Header2 </h2> <h3> Header3 </h3>" +
                    "<h4> Header4 </h4> <h5> Header5 </h5> <h6> Header6 </h6> Text <br> separated by" +
                    " <br> breaklines. <b> Bold text </b> <i> Italic text </i> <p> Text in paragraph " +
                    "</p> <s> Strikethrough text </s> <mark> Highlighted text </mark> and " +
                    "<a href=\"https://www.google.com/\"> text with link </a> and just <u> Underlined" +
                    " text </u> unordered list: <ul> <li>first item</li> <li>second item</li>" +
                    " <li>third item</li> </ul> Text outside markup tags",

Reply:
Good point, changed it.